src/app.js

```diff
@@ -1,3 +1,22 @@
 'use strict';
 
-// Write your code here
+import readline from 'readline';
+import { generateRandomNumber } from './modules/generateRandomNumber';
+import { checkIsValidUserInput } from './modules/checkIsValidUserInput';
+import { getBullsAndCows } from './modules/getBullsAndCows';
+
+const terminal = readline.createInterface(process.stdin, process.stdout);
+
+const numberToGuess = generateRandomNumber();
+
+terminal.question('Guess the number', (userNumber) => {
+  if (!checkIsValidUserInput(userNumber)) {
+    return;
+  }
+
+  if (userNumber === numberToGuess) {
+    terminal.close();
```

Comment:
After closing the terminal, it might be a good idea to inform the user that they have guessed the correct number. Consider adding a message to congratulate the user.